sycl/source/detail/compiler.hpp

```diff
@@ -44,8 +44,11 @@
 /// PropertySetIO.h
 #define __SYCL_PROPERTY_SET_SPEC_CONST_DEFAULT_VALUES_MAP                      \
   "SYCL/specialization constants default values"
+/// TODO: remove req mask when sycl devicelib online linking path is removed.
 /// PropertySetRegistry::SYCL_DEVICELIB_REQ_MASK defined in PropertySetIO.h
 #define __SYCL_PROPERTY_SET_DEVICELIB_REQ_MASK "SYCL/devicelib req mask"
+/// PropertySetRegistry::SYCL_DEVICELIB_BF16_TYPE defined in PropertySetIO.h
+#define __SYCL_PROPERTY_SET_DEVICELIB_BF16_TYPE "SYCL/BF16 devicelib type"
```

Comment:
Do we anticipate more of "special" libraries like bf16? If so, I would suggest to name the property set in a more generic way. See my comments below about use of `selectBF16Devicelib` which should give a possible idea of a generalization here.

Reply:
Hi, @AlexeySachkov 
I didn't anticipate any more of "special" libraries like bf16. For other devicelib files, we intend to link them offline(during compilation) to reduce runtime complexity. However, it is easy to make the whole mechanism to be more general, do you think we need to do this here?
Thanks very much.

Reply:
> I didn't anticipate any more of "special" libraries like bf16. For other devicelib files, we intend to link them offline(during compilation) to reduce runtime complexity.

In that case there is no need for early and potentially unnecessary generalization which will complicate everything, we can think of it once we have an actual use case for it